### PR TITLE
feature/COR-1784_COR-1801_vaccinations-modifications

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard30**
Combination branch for branches [COR-1784](https://github.com/minvws/nl-covid19-data-dashboard/tree/feature/COR-1784_vaccinations-national-2) and [COR-1801](https://github.com/minvws/nl-covid19-data-dashboard/tree/feature/COR-1801_vaccinations-modifications-municipalities)

### CODE REVIEW

File: packages/app/src/components/kpi/components/kpi-content.tsx [hunk 1]
```diff
@@ -36,6 +42,7 @@ export const KpiContent = ({ title, description, value, bar, birthyear, differen
             : description
         }
       />
+      <Metadata {...metadata} isTileFooter marginBottom={space[2]} />
```
Comment (VWSCoronaDashboard29) [inline packages/app/src/components/kpi/components/kpi-content.tsx:45]: This should be optional

**Review by VWSCoronaDashboard29 — CHANGES_REQUESTED** [review]

File: packages/app/schema/archived_nl/vaccine_campaigns_archived_20231004.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (VWSCoronaDashboard30) [inline packages/app/schema/archived_nl/vaccine_campaigns_archived_20231004.json:1]: @VWSCoronaDashboard29 We need to change the naming for this one. It should not contain the archived date in the title nor name of the file.

File: packages/app/schema/archived_nl/vaccine_coverage_per_age_group_archived_20231012.json
Comment (VWSCoronaDashboard30) [inline packages/app/schema/archived_nl/vaccine_coverage_per_age_group_archived_20231012.json:1]: same here @VWSCoronaDashboard29

File: packages/app/schema/archived_nl/vaccine_coverage_per_age_group_estimated_autumn_2022.json
Comment (VWSCoronaDashboard30) [inline packages/app/schema/archived_nl/vaccine_coverage_per_age_group_estimated_autumn_2022.json:1]: same here @VWSCoronaDashboard29

File: packages/common/src/feature-flags/features.ts [hunk 3]
```diff
@@ -4,13 +4,13 @@ export const features: Feature[] = [
   {
```
Comment (VWSCoronaDashboard30) [inline packages/common/src/feature-flags/features.ts:4]: This one is not fixed yet @VWSCoronaDashboard29

File: packages/cli/src/schema/schema-info.ts [hunk 4]
```diff
@@ -39,7 +42,7 @@ export function getSchemaInfo(jsonDirectory: string = defaultJsonDirectory): Sch
       files: getFileNames(archivedFileList, /^GM[0-9]+.json$/),
       basePath: path.join(jsonDirectory, 'archived'),
       customValidations: [
-        createChoroplethValidation(path.join(defaultJsonDirectory, 'archived', 'GM_COLLECTION.json'), 'gmcode', ['vaccine_coverage_per_age_group']),
+        createChoroplethValidation(path.join(defaultJsonDirectory, 'archived', 'GM_COLLECTION.json'), 'gmcode', ['vaccine_coverage_per_age_group_archived_202310xx']),
```
Comment (VWSCoronaDashboard30) [inline packages/cli/src/schema/schema-info.ts:45]: @VWSCoronaDashboard29 Can you fix this one as well or should i take care of it?

File: packages/app/schema/archived_nl/vaccine_campaigns_archived_20231004.json (hunk 2 above)
Comment (VWSCoronaDashboard29) [reply packages/app/schema/archived_nl/vaccine_campaigns_archived_20231004.json:1]: What do you want to change the filename to?

File: packages/app/schema/archived_nl/vaccine_coverage_per_age_group_archived_20231012.json
Comment (VWSCoronaDashboard29) [reply packages/app/schema/archived_nl/vaccine_coverage_per_age_group_archived_20231012.json:1]: Same question as in previous comment https://github.com/minvws/nl-covid19-data-dashboard/pull/4887#discussion_r1342639055

File: packages/cli/src/schema/schema-info.ts (hunk 4 above)
Comment (VWSCoronaDashboard29) [reply packages/cli/src/schema/schema-info.ts:45]: See https://github.com/minvws/nl-covid19-data-dashboard/pull/4887#discussion_r1342651681

File: packages/common/src/feature-flags/features.ts (hunk 3 above)
Comment (VWSCoronaDashboard29) [reply packages/common/src/feature-flags/features.ts:4]: Should be fixed now

**Review by VWSCoronaDashboard29 — APPROVED** [review]

**Pull request merged**